libs/wasm-transform/examples/usage.rs

```diff
@@ -26,8 +29,13 @@ fn overwrite() {
 
     let req = transform::Request {
         data: sample_json.into(),
-        path: "hello".to_string(),
         value: r#""baz""#.to_string(),
+        paths: vec![DetectiveStepResultMatch {
+            path: "hello".to_string(),
+            value: "".to_string().into_bytes(),
```

Comment:
Stick a comment after this `value: "".to_string().into_bytes(),` (//) - pointing out that this is not needed - just a rust default set.